include/bitbuffer.h

```diff
@@ -14,7 +14,7 @@
 
 #include <stdint.h>
 
-#define BITBUF_COLS 256 // Number of bytes in a column
+#define BITBUF_COLS 512 // Number of bytes in a column
```

Comment:
Can we make that 400 columns (if I check correctly you need 392) and add a comment that "wmbus needs at least 392"

Reply:
do we need so low limits? It is not running on embedded system, so 256B of memory is ok I hope :)

Reply:
actually +256B times BITBUF_ROWS, and we do aim to run on low-end embedded (e.g. old routers) ;)

Reply:
IMHO then we need at least ~437B  (( (256+16*2+3)*12)/8 ), when the data len will be 256 + 16*2 CRC + 3B